.github/workflows/main.yml

```diff
@@ -0,0 +1,146 @@
+name: OpenDingux_buildroot
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/arcnor/retro-toolchain/buildroot
```

Comment:
This is a Docker image used for building that comes from https://github.com/Arcnor/RetroHandhelds-Docker. I think it will be nice if that repo was also part of OpenDingux, so everything is "self-contained".

Reply:
How can I make it part of OpenDingux?

Reply:
Just fork the repo, it will automatically create the docker images on the same repository (you can see them on mine under the "Packages" section on the lower right side on GH), as I made it "account agnostic". If everything works correctly, you should have 3 packages on the forked repo, then we can change this PR to use those new Docker images instead.

Reply:
I forked to https://github.com/OpenDingux/RetroHandhelds-Docker